app/src/main/java/com/cornellappdev/uplift/ui/screens/ReportIssueScreen.kt

```diff
@@ -0,0 +1,131 @@
+package com.cornellappdev.uplift.ui.screens
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.cornellappdev.uplift.ui.components.general.UpliftTopBarWithBack
+import com.cornellappdev.uplift.ui.components.reporting.ReportDescription
+import com.cornellappdev.uplift.ui.components.reporting.ReportDropdown
+import com.cornellappdev.uplift.ui.components.reporting.SubmitButton
+
+/**
+ * ReportIssueScreen is a composable that displays the report issue screen where users can report
+ * an issue with the app.
+ */
+@Composable
+fun ReportIssueScreen(
+    onSubmit: (issue: String, gym: String, description: String) -> Unit,
+    onBack: () -> Unit
+) {
+    var selectedIssue by remember { mutableStateOf("Choose an option ...") }
```

Comment:
This may not be a question you can answer, but do you know if this screen is supposed to have its viewmodel or if all report screens are to share a singular overarching viewmodel?

If we aim to have a specific viewmodel for this screen (probably what I'd recommend), we should add a TODO to replace the calls to `onSubmit` and `onBack` with the corresponding VM function calls. In that case, `ReportIssueScreen` would use the `ReportIssueViewModel` and call, like, `reportIssueViewModel.onSubmit(...)`

Reply:
I don't know for certain, but I believe that this is the main report screen, with the other screen being the report submission success screen which shouldn't really need to interact with the backend, so I would probably guess that there would be one specific VM dedicated to this screen. I'll add the TODO for the onSubmit param. I was intending for the onBack to be a navigation function from the nav wrapper that triggers when the back arrow is clicked, so not sure if there will be a VM associated with it.